data_scientist_skills/skill_extraction.py

```diff
@@ -10,9 +10,14 @@ def remove_more_stopwords(text):
     clean_text.replace(',', '')
     return clean_text
 
-def tdidf(X, min_df=0.1, max_df=0.6, results_df=False):
+def tdidf(X, min_df=0.1, max_df=0.6, results_df=False, transform=False):
     """Performs TD-IDF on pd Series. Results_df=True to return table of results. """
     tfidf = TfidfVectorizer(ngram_range=(1,2), min_df=min_df, max_df=max_df)
+
+    if transform:
+        # tfidf.fit(X)
```

Comment:
So if transform is set to True the function exits and gives the initialized but unfit vectorizer? I trust that it's useful but I don't follow this option

Reply:
Yeah for some reason I couldn't fit within this function, and then use it to transform the input outside of this function. So simply getting the same model for use on the input

Reply:
I see now that it's used that way in the init_vectorizer